src/ui/simulator/windows/options/adequacy-patch/adequacy-patch-options.cpp

```diff
@@ -0,0 +1,570 @@
+/*
+** Copyright 2007-2018 RTE
+** Authors: Antares_Simulator Team
+**
+** This file is part of Antares_Simulator.
+**
+** Antares_Simulator is free software: you can redistribute it and/or modify
+** it under the terms of the GNU General Public License as published by
+** the Free Software Foundation, either version 3 of the License, or
+** (at your option) any later version.
+**
+** There are special exceptions to the terms and conditions of the
+** license as they are applied to this software. View the full text of
+** the exceptions in file COPYING.txt in the directory of this software
+** distribution
+**
+** Antares_Simulator is distributed in the hope that it will be useful,
+** but WITHOUT ANY WARRANTY; without even the implied warranty of
+** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+** GNU General Public License for more details.
+**
+** You should have received a copy of the GNU General Public License
+** along with Antares_Simulator. If not, see <http://www.gnu.org/licenses/>.
+**
+** SPDX-License-Identifier: licenceRef-GPL3_WITH_RTE-Exceptions
+*/
+
+#include "adequacy-patch-options.h"
+#include <wx/sizer.h>
+#include <wx/checkbox.h>
+#include <wx/stattext.h>
+#include <wx/statline.h>
+#include "../../../toolbox/create.h"
+#include "../../../toolbox/resources.h"
+#include "../../../application/study.h"
+#include "../../../application/menus.h"
+#include "../../../windows/message.h"
+#include <ui/common/component/panel.h>
+#include <antares/logs.h>
+
+using namespace Yuni;
+
+namespace Antares
+{
+namespace Window
+{
+namespace Options
+{
+static void SubTitle(wxWindow* parent, wxSizer* sizer, const wxChar* text, bool margintop = true)
```

Comment:
For now, you don't use **margintop** argument, we should think about removing it